src/components/SearchBar.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const SearchBar = ({ input, onChange }) => {
+  return (
+    <div className="search-bar w-full max-w-xl">
+      <label htmlFor="query" className="label sr-only">
+        Food Name
+      </label>
+      <input
+        type="text"
```

Comment:
Using `type="search"` might be a better option instead of `"text"`. Just remember that it adds a clear button (a little `x` on the right side of the input), which you might not want. In that case, you can disregard my comment.